Cargo.toml

```diff
@@ -23,7 +23,7 @@ num-traits = "0.2.6"
 async-smtp = "0.3"
 email = { git = "https://github.com/deltachat/rust-email", branch = "master" }
 lettre_email = { git = "https://github.com/deltachat/lettre", branch = "master" }
-async-imap = "0.3.1"
+async-imap = { path = "../../async-imap" }
```

Comment:
is this a residual change from your local testing?

Reply:
This PR is waiting for https://github.com/async-email/async-imap/pull/36 to be merged. After that, `Cargo.toml` should be reverted and `Cargo.lock` updated.

Reply:
cc @dignifiedquire